app/helpers/application_helper.rb

```diff
@@ -93,10 +95,11 @@ def data_asset_display_for_plate(facts)
         else
           aliquotType = empty_well_aliquot_type
         end
+
         memo[location] = {
           title: "#{asset.short_description}",
           cssClass: aliquotType,
-          url: asset_path(asset)
+          url: Rails.application.routes.url_helpers.asset_path(asset)
```

Comment:
This LOC was spewing out warnings, as asset_path in this context was actually for images and the like.